docs/creating-components.md

```diff
@@ -211,3 +211,47 @@ import CustomButton from './CustomButton';
 ```
 
 For more details and advanced theming features and recommendations, see our [Theming Guide](./theming.md).
+
+## Customizing Sandstone skin at Run-Time
```

Comment:
I think we could use `Runtime` instead of `Run-Time` for consistency. WDYT?

Reply:
The previous title is `## Customizing Components at Design-Time` that's why I put the same format :)